hathor/utils/yaml.py

```diff
@@ -0,0 +1,23 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import Dict
+
+import yaml
+
+
+def dict_from(*, filepath: str) -> Dict:
```

Comment:
Maybe `Dict[str, Any]`?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/593/commits/48715e3d8a37dcb59ff1cbb2c3ce848906b8c42a

Reply:
Thanks!